tests/flow/constraint_utils.py

```diff
@@ -85,13 +85,24 @@ def create_mandatory_constraint(g, entity_type, lbl, *props, sync=False):
     return create_constraint(g, "MANDATORY", entity_type, lbl, *props, sync=sync)
 
 def create_unique_node_constraint(g, lbl, *props, sync=False):
-    # create exact-match index
-    create_node_exact_match_index(g, lbl, *props, sync=False)
+    # create exact-match index(es)
+    for p in props:
```

Comment:
breaking change

Reply:
Indeed this is a breaking change, I think this is the right path to take here.